subsystem/tests/helpers/mod.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://spdx.org/licenses/MIT
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author(s): L. Kuklinek
+
+static INIT: std::sync::Once = std::sync::Once::new();
+
+pub fn init_test_runtime() -> tokio::runtime::Runtime {
+    INIT.call_once(|| logging::init_logging::<&std::path::Path>(None));
```

Comment:
Does this reset for every individual test, give that it's declared globally static?

Reply:
No it's only once per test binary run and one run of test binary may run multiple tests. Calling `init_logging` multiple times (e.g. for every test) causes a panic, that's why it's only initialised once using `std::sync::Once`.

Reply:
Perfect!